src/pages/PageOrgchart.vue

```diff
@@ -107,7 +106,35 @@ export default {
       looseTrace: '',
       previewProfileQuery: PREVIEW_PROFILE,
       desktopView: false,
-      baseState: 'normal',
+
+      expanded: {},
+      collapsed: null,
+      subscribers: {},
+    };
+  },
+  provide() {
+    const self = this;
```

Comment:
Maybe we can call this `orgChartPage` I love the nostalgia here, but my first though was why do we need this.

Reply:
Haha I felt nostalgic writing it! Fair enough, I'll change it

Reply:
Unfortunately we need it because Vue binds functions given through `provide` to the component they're being `injected` into